src/lib/components/ProgressBar.svelte

```diff
@@ -4,7 +4,9 @@
   // Html default is 1 anyway
   // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress?retiredLocale=ca#attr-max
   export let max = 1;
-  export let value: number;
+  // `value` becomes now optional
```

Comment:
This comment was useful for the review but not worth keeping, right?
Otherwise you might as well write on each of the props that they are optional. Or am I missing something?

Reply:
I initially thought of leaving it. But yes, it adds no value.